<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evolution</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: darkblue;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="picNW">
      <property name="styleSheet">
       <string notr="true">background-color: yellow;</string>
      </property>
      <property name="text">
       <string>picNW</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="picNE">
      <property name="styleSheet">
       <string notr="true">background-color: blue;</string>
      </property>
      <property name="text">
       <string>picNE</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="picN">
      <property name="styleSheet">
       <string notr="true">background-color: cyan;</string>
      </property>
      <property name="text">
       <string>picN</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="picW">
      <property name="styleSheet">
       <string notr="true">background-color: magenta;</string>
      </property>
      <property name="text">
       <string>picW</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="picSW">
      <property name="styleSheet">
       <string notr="true">background-color: lime;</string>
      </property>
      <property name="text">
       <string>picSW</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="picC">
      <property name="styleSheet">
       <string notr="true">background-color: white</string>
      </property>
      <property name="text">
       <string>picC</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="picE">
      <property name="styleSheet">
       <string notr="true">background-color: pink;</string>
      </property>
      <property name="text">
       <string>picE</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="picS">
      <property name="styleSheet">
       <string notr="true">background-color: salmon;</string>
      </property>
      <property name="text">
       <string>picS</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="picSE">
      <property name="styleSheet">
       <string notr="true">background-color: red;</string>
      </property>
      <property name="text">
       <string>picSE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string extracomment="The usual items"/>
    </property>
    <property name="statusTip">
     <string extracomment="status tip">The usual...</string>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionFullScreen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEvolution">
    <property name="title">
     <string>&amp;Evolution</string>
    </property>
    <addaction name="actionBreed"/>
    <addaction name="actionKill"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEvolution"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string extracomment="comment for going home">Going home...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="text">
    <string>Full&amp;Screen</string>
   </property>
   <property name="statusTip">
    <string>show Evolution in full screen</string>
   </property>
  </action>
  <action name="actionBreed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Breed</string>
   </property>
  </action>
  <action name="actionKill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Kill</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
